lib/t/matches.ex

```diff
@@ -20,6 +20,8 @@ defmodule T.Matches do
   @pubsub T.PubSub
   @topic "__m"
 
+  @match_expiration_duration 172_800
```

Comment:
What's 172_800? Also values like this shouldn't be hardcoded but passed as args.